uktrade/data-hub-frontend
# Feature/investments profiles filters

## Description of change

Adds the _Asset class of interest_ and _Company name_ to the `/investments/profiles` page. The filters are hidden behind the `capital-investments-filters` feature flag.

## Test instructions

1. Enable the `capital-investments-filters` feature flag
1. Go to the `/investments/profiles` page
2. There should the two new filters
3. Use the filters
4. The results should reflect the filters
